blueprints/posts-via-wp-cli/blueprint.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+    "meta": {
```

Comment:
Thanks for providing the `meta` context. Are these fields the same that go in plugin headers, @bph?

Reply:
@flexseth they're not, see https://github.com/adamziel/blueprints/blob/trunk/CONTRIBUTING.md

Reply:
> @flexseth they're not, see https://github.com/adamziel/blueprints/blob/trunk/CONTRIBUTING.md

Ok, the docs say metadata is _required_ but I've seen plenty of functioning blueprints without meta.

Is the requirement part of the V2 spec? Or should the "requirement" part be removed?

### Rewording?
While they aren't required, maybe it can be worded to explain why the metadata is so important?

Reply:
The requirement is just for the community space submissions so we can render a gallery. A rewording could be useful indeed.